test/e2e/backup_restore_test.go

```diff
@@ -195,6 +197,14 @@ func assertNumEntries(cacheName, pod string, numEntries int, client ispnclient.H
 	if rsp.StatusCode != http.StatusOK {
 		panic(fmt.Sprintf("Unexpected response code %d", rsp.StatusCode))
 	}
+
+	body, err := ioutil.ReadAll(rsp.Body)
+	tutils.ExpectNoError(err)
+	numRead, err := strconv.ParseInt(string(body), 10, 64)
+	tutils.ExpectNoError(err)
+	if int(numRead) != numEntries {
```

Comment:
@ryanemerson I added numEntries validation and got 0 cache entries to return. Could you have a look at this problem.

Reply:
Skip this validation as now can't be passed